Medicare/src/DatabaseAccessLayer/DAO.java

```diff
@@ -71,420 +74,238 @@ public ResultSet getRs() {
     public void setRs(ResultSet rs) {
         this.rs = rs;
     }
-    
-     public void DoConnect( ) {
 
-       try {
+    public String getLoggedInUser() {
+        return loggedInUser;
+    }
+
+    public void setLoggedInUser(String loggedInUser) {
+        this.loggedInUser = loggedInUser;
+    }
+
+    public void DoConnect() {
+
+        try {
             InputStream bdl = this.getClass().getResourceAsStream("/resources/dbconfig.properties");

```

Comment:
define the string as a constant on top of the file:
/resources/dbconfig.properties